crates/alpen-ee/common/src/traits/storage/batch.rs

```diff
@@ -0,0 +1,80 @@
+use async_trait::async_trait;
+
+use crate::{Batch, BatchId, BatchStatus, Chunk, ChunkId, ChunkStatus, StorageError};
+
+/// Storage for Batches and Chunks
+///
+/// A batch is a contiguous group of blocks. One batch corresponds to one Account Update Operation
+/// sent to OL, and batch size should typically be limited by DA size constraints.
+///
+/// Chunks are subsets of blocks in a batch, and units for proving ee chain execution, typically
+/// limited by prover constraints.
+///
+/// An Batch will consist of 1 or more chunks.
+/// These chunks can be generated before or after updates, but the chunks must fit inside Batch
```

Comment:
```suggestion
/// These chunks can be generated before or after updates, but the chunks must fit inside batch
```
Can you clarify the relation to update here?